topology/University of Wisconsin/EHT/EHT.yaml

```diff
@@ -0,0 +1,34 @@
+GroupDescription: EHT OSPool Resources
+GroupID: 1303
+Production: true
+SupportCenter: Self Supported
+Resources:
+  EHT-EP:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: OSG1000003
+          Name: Brian Lin
+        Secondary:
+          ID: OSG1000002
+          Name: Matyas Selmeci
+        Tertiary:
+          ID: OSG1000007
+          Name: Carl Edquist
+      Security Contact:
+        Primary:
+          ID: OSG1000003
+          Name: Brian Lin
+        Secondary:
+          ID: OSG1000002
+          Name: Matyas Selmeci
+    Description: OSPool A100 for EHT
+    FQDN: tiger2002.chtc.wisc.edu
```

Comment:
Let's give it a nicer fake FQDN (maybe this ends up being a different node in the future): eht-ep.chtc.wisc.edu maybe

Reply:
:+1:

Reply:
updated